DfE.FindInformationAcademiesTrusts/Services/Export/ExportService.cs

```diff
@@ -41,33 +41,37 @@ public async Task<byte[]> ExportAcademiesToSpreadsheetAsync(string uid)
             "Type",
             "Rural or Urban",
             DateJoined,
-            "Previous Ofsted Rating",
-            BeforeOrAfterJoiningHeader,
-            "Date of Previous Ofsted",
             "Current Ofsted Rating",
             BeforeOrAfterJoiningHeader,
             "Date of Current Ofsted",
+            "Previous Ofsted Rating",
+            BeforeOrAfterJoiningHeader,
+            "Date of Previous Ofsted",
             "Phase of Education",
             AgeRange,
             "Pupil Numbers",
             "Capacity",
             "% Full",
-            "Pupils eligible for Free School Meals"
+            "Pupils eligible for Free School Meals",
+            "LA average pupils eligible for Free School Meals",
+            "National average pupils eligible for Free School Meals",
         };
 
         var trustSummary = await trustRepository.GetTrustSummaryAsync(uid);
         var academiesDetails = await academyRepository.GetAcademiesInTrustDetailsAsync(uid);
         var academiesOfstedRatings = await academyRepository.GetAcademiesInTrustOfstedAsync(uid);
         var academiesPupilNumbers = await academyRepository.GetAcademiesInTrustPupilNumbersAsync(uid);
         var academiesFreeSchoolMeals = await academyRepository.GetAcademiesInTrustFreeSchoolMealsAsync(uid);
+        var academiesFreeSchoolMealsAverages = await academyService.GetAcademiesInTrustFreeSchoolMealsAsync(uid);
```

Comment:
It would be good to use this object from the service for all free school meal data as it would reduce the cognitive load of having two very similar object - there shouldn't be anything we're using from the repo object in the export that isn't available in the service object